x/upgrade/internal/types/proposal.go

```diff
@@ -11,6 +11,7 @@ const (
 	ProposalTypeSoftwareUpgrade       string            = "SoftwareUpgrade"
 	ProposalTypeCancelSoftwareUpgrade string            = "CancelSoftwareUpgrade"
 	DefaultCodespace                  sdk.CodespaceType = "upgrade"
+	FlagUnsafeSkipUpgrade             string            = "unsafe-skip-upgrade"
```

Comment:
this should only be declared in the `cli` package

Reply:
Fixed it